src/components/SinglePropertyDetail.tsx

```diff
@@ -289,22 +292,16 @@ const SinglePropertyDetail = ({
         </a>
       </p>
 
-      <h3 className="font-bold mb-2 py-2 heading-xl">
-        Ways to transform the lot
-      </h3>
-      <p className="mb-4">
-        To see different ways in which you might transform this property, see
-        <a
-          href="/transform-property"
-          target="_blank"
-          rel="noopener noreferrer"
-          className="link"
-        >
-          {" "}
-          our <Tree className="inline h-6 w-6" aria-hidden="true" /> Transform a
-          Property page.
-        </a>
-      </p>
+      <h3 className="font-bold mb-2 py-2 heading-xl">Transform a Property</h3>
+      <a href="/transform-property">
```

Comment:
could we update this to next.js's `<Link>` component? using `Link` to one of our own pages utilizes prefetching to speed up page loads! you can read more here: https://nextjs.org/docs/pages/api-reference/components/link#prefetch